lasagne/tests/test_init.py

```diff
@@ -15,55 +15,137 @@ def test_normal():
 
     sample = Normal().sample((100, 200))
     assert -0.001 < sample.mean() < 0.001
+    assert 0.009 < sample.std() < 0.011
 
 
-def test_constant():
-    from lasagne.init import Constant
+def test_uniform_range_as_number():
+    from lasagne.init import Uniform
 
-    sample = Constant(1.0).sample((10, 20))
-    assert (sample == 1.0).all()
+    sample = Uniform(1.0).sample((300, 400))
+    assert sample.shape == (300, 400)
+    assert -1.1 < sample.min() < -0.9
+    assert 0.9 < sample.max() < 1.1
```

Comment:
I'd do `assert -1.0 <= sample.min() < -0.9` and `assert 0.9 < sample.max() <= 1.0`. We can check the bounds more strictly.